src/BaseResponse.php

```diff
@@ -235,10 +235,19 @@ public function send() {
 		}
 
 		// messages cookie
-		if( isset($this->messages) ) {
+		if( isset($this->messages) && !empty($this->messages) ) {
```

Comment:
I don't think the `isset($this->messages)` is necessary because `!empty()` will check the same thing as the `$messages` property is already declared

Reply:
TBH passing messages in cookies is fucking demented.

What happens if we just remove this?